packages/lesswrong/lib/collections/conversations/schema.ts

```diff
@@ -50,6 +51,7 @@ const schema: SchemaType<DbConversation> = {
     insertableBy: ['members'],
     editableBy: ['admins'],
     optional: true,
+    hidden: !['LessWrong', 'AlignmentForum'].includes(forumTypeSetting.get())
```

Comment:
This was an option that appeared as an "AF" checkbox in the conversation options - I assume it should not appear on forums that are not LW or AF.